### DISCUSSION TITLE
Svelte Adapter LLL

### DISCUSSION THREAD

**Discussion by mledl** [ivanhofer/typesafe-i18n#642] [Q&A] [answered]
Does the Svelte Adapter support the `LLL` string translation as well? If I want to use the store provided by the Svelte Adapter and the `LLL` utility, do I really have to call both `setLocale` methods?

**Comment by ivanhofer** [comment, marked as answer]
Sure, take a look: https://github.com/ivanhofer/typesafe-i18n-demo-svelte/blob/master/src/i18n/i18n-util.ts#L21
You probably already use it in the project, unless you removed that function, I wrote a while ago😜

You need to create instances manually if you want to use it.
